certified-connectors/HubPlanner/apiDefinition.swagger.json

```diff
@@ -0,0 +1,644 @@
+                  "title": "Start date of booking"
+                },
+                "end": {
+                  "type": "string",
+                  "description": "format YYYY-MM-DD",
+                  "title": "End date of booking",
+                  "format": ""
+                },
+                "project": {
+                  "type": "string",
+                  "description": "The project _id you want to schedule (From Hub Planner)",
+                  "title": "Project ID"
+                }
+              },
+              "required": [
+                "end",
+                "start",
+                "project",
+                "resource"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/timeentry": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Create Time Entry",
+        "description": "Create Time Entry",
+        "operationId": "CreateTimeEntry",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "resource": {
+                  "type": "string",
+                  "description": "Resource _id of the project from Hub Planner",
+                  "title": "Resource ID"
+                },
+                "project": {
+                  "type": "string",
+                  "description": "Project _id of the project from Hub Planner",
+                  "title": "Project ID"
+                },
+                "date": {
+                  "type": "string",
+                  "description": "Format YYYY-MM-DD",
+                  "title": "Date of time entry"
+                },
+                "minutes": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Value of time entry in minutes, e.g. passing 60 is equal to 1 hr",
+                  "title": "Value (Minutes)"
+                }
+              },
+              "required": [
+                "date",
+                "project",
+                "resource",
+                "minutes"
+              ]
+            }
+          }
+        ]
+      }
+    }
+  },
+  "definitions": {},
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {
+    "API Key": {
+      "type": "apiKey",
+      "in": "header",
+      "name": "Authorization"
+    }
+  },
+  "security": [
+    {
+      "API Key": []
+    }
+  ],
+  "tags": [],
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://hubplanner.com"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://hubplanner.com/privacy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Resource Management"
```

Comment:
Hello @fikatown 

Could you please fix the below error? kindly mention only allowed values. 

##[error]The connector category is invalid. Available categories are: [AI, Business Management, Business Intelligence, Collaboration, Commerce, Communication, Content and Files, Finance, Data, Human Resources, Internet of Things, IT Operations, Lifestyle and Entertainment, Marketing, Productivity, Sales and CRM, Security, Social Media, Website]. ValuePath: /x-ms-connector-metadata/2/propertyValue